### PR TITLE
Remove Mappings Language

### PR DESCRIPTION
**Pull request by HeikoKlare**
As agreed upon in a developer meeting, this PR removed the Mappings language due to its currently unusable state and our inability to develop it to a usable state and then maintain it.

This PR also fixes the standalone setup of the Reactions language, which did only work by now because with the Mappings language was always initialized when using the Reactions language, and the Mappings then initialized the Reactions properly.

### CODE REVIEW

File: bundles/tools.vitruv.dsls.reactions/workflow/generate.mwe2 [hunk 1]
```diff
@@ -18,7 +18,7 @@ Workflow {
 		from = "http://www.eclipse.org/emf/2002/Ecore"
 		to = "platform:/resource/org.eclipse.emf.ecore/model/Ecore.ecore"
 	}
-
+
```
Comment (JanWittler) [inline bundles/tools.vitruv.dsls.reactions/workflow/generate.mwe2:21]:
```suggestion

```

unnecessary file change

**Review by JanWittler — APPROVED** [review]
Thanks.
Should we maybe set a tag /create a stale branch to find the latest commit containing the Mappings language easier - in case it might be reactivated in the future?

**Comment by JanWittler** [comment]
The organisation's Readme and the Wiki need to be updated after merging this PR.

**Comment by HeikoKlare** [comment]
Yes, make finding the language easier makes sense. I would prefer to have a tag, but do not have a strong opinion on that either.

I will update readme and wiki as soon as I merge the PR.

**Comment by HeikoKlare** [comment]
Readme and wiki have been updated.

**Comment by HeikoKlare** [comment]
I have added an annotated tag `mappings-language` to the last commit containing the Mappings language.

**Pull request merged**